### ISSUE TITLE
Server-only install option

### ISSUE THREAD

**Issue by risto-liftoff** [DefectDojo/godojo#84]
Thanks for merging the PRs I opened. I'm wondering if you'd be open to me creating a PR to add a server-only install option that doesn't make any modifications to the database. Because my company has the expectation that any server can be thrown away and automatically replaced, I made the changes in a fork and added a server-only install option.

**Comment by mtesauro** [comment]
@risto-liftoff  Interesting use-case - godojo was really created to replace an existing bash-based installer that only did the initial setup so that was never really considered.

Do you have links to how you've done that in your fork? I'm curious how you implemented that case.

**Comment by risto-liftoff** [comment]
Yeah, it's the last 2 commits on this branch: https://github.com/risto-liftoff/godojo/tree/separate-setup-ubuntu.

I only implemented it for Ubuntu, but I can add my changes to RHEL and the template OS files. And instead of having the config option be `Install.DB.DontModify`, I'd probably change it to `Install.ServerOnly`.

**Comment by mtesauro** [comment]
Unfortunately, godojo is in the process of being deprecated so we're not going to be making any substantive changes going forward.

For details on deprecating godojo - see the GH discussion at:
https://github.com/DefectDojo/django-DefectDojo/discussions/11199
in the "Future Deprecation: Iron installs via godojo" section.
